TODO.md

```diff
@@ -58,10 +46,10 @@ Steps to standardize a commodity table (~FI\_COMM)
   - [x] Csets: declarations are inherited until the next one is encountered
   - [x] Region: By default, it is applied to all regions of the model when not specified
     - [x] Get hold of regions from  ~BookRegions\_Map, can use comma-separated list here
-- [x] LimType: When not specified, the default is LO for all but MAT commodities (Csets column) with a default of FX
-- [x] CTSLvl: When not specified, the default is ANNUAL
-- [x] PeakTS: If not specified the default is ANNUAL
-- [x] Duplicate rows with comma-separated values (Csets, Region, PeakTS), each row getting only one of the values.
+- [x] ?LimType: When not specified, the default is LO for all but MAT commodities (Csets column) with a default of FX
+- [x] ?CTSLvl: When not specified, the default is ANNUAL
+- [x] ?PeakTS: If not specified the default is ANNUAL
+- [x] ?Duplicate rows with comma-separated values (Csets, Region, PeakTS), each row getting only one of the values.
```

Comment:
What do these question marks indicate? Are these the ones you're checking?

Reply:
Yes, left a mark to double-check 🤣